src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -138,18 +129,22 @@ public static String parseMeetingType(String type) throws ParseException {
     }
 
     /**
-     * Parses {@code date} into an {@code Date} and returns it. Leading and trailing whitespaces will be
+     * Parses {@code meetingDate} into an {@code MeetingDate} and returns it. Leading and trailing whitespaces will be
      * trimmed.
      *
-     * @throws ParseException if the format of date is wrong.
+     * @throws ParseException if the format of meetingDate is wrong.
      */
-    public static Date parseDate(String date) throws ParseException {
+    public static MeetingDate parseDate(String date) throws ParseException {
         String trimmedTime = date.trim();
         SimpleDateFormat formatter = new SimpleDateFormat("dd-MM-yyyy");
         try {
-            java.util.Date dateObject = formatter.parse(trimmedTime);
+            Date dateObject = formatter.parse(trimmedTime);
+            Date currentActualDate = Calendar.getInstance().getTime();
+            if (dateObject.before(currentActualDate)) {
```

Comment:
Looks good but maybe you can consider if the meeting date input is unrealistic (e.g 01-01-2099) other than that, this LGTM! Well done!